classes/helper.php

```diff
@@ -98,11 +98,13 @@ public static function enable_notifications() : string {
         if (get_capability_info('local/csp:seenotifications')) {
             $cansee = has_capability('local/csp:seenotifications', $PAGE->context, $USER->id);
         }
-        if (!$cansee | ($conf->notifications_enable_enforced + $conf->notifications_enable_reported == 0)) {
+        $notificationsenableenforced = $conf->notifications_enable_enforced ?? 0;
```

Comment:
I think we run this on some older sites before php7 before ?? was available, so I think it's easier to fix this here rather than make a new stable branch

Reply:
Ok, changed this to use `!isset` instead.